src/app/pages/Explore.tsx

```diff
@@ -111,6 +112,7 @@ const Explore: FC<ExploreProps> = ({ assetSlug }) => {
               pathname: '/swap',
               search: `from=${assetSlug ?? ''}`
             }}
+            testID={ExploreSelectors.SwapButton}
```

Comment:
according to the style guide `testID` should be the last prop (event after event handlers)